assets/css/style.css

```diff
@@ -0,0 +1,165 @@
+@font-face {
+  font-family: Inter;
+  src: url(../fonts/Inter/Inter-VariableFont_slnt\,wght.ttf);
+}
+
+* {
+  font-family: 'Inter', Arial, Helvetica, sans-serif;
+}
+
+body {
+  padding: 0;
+  margin: 0;
+  background-color: #fff;
+  color: #1e2823;
+}
+
+.toolbar {
+  display: flex;
+  justify-content: space-between;
+  padding: 20px 5%;
+  position: fixed;
+  top: 0;
+  width: 90%;
+  background-color: #fff;
+  height: 45px;
+}
+
+.logo,
+.toggle-menu {
+  padding: 10px 12px;
+}
+
+.logo a {
+  font-weight: 600;
+  font-size: 20px;
+  color: inherit;
+  text-decoration: none;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  margin-top: 90px;
+  background-image: url(../images/header-illsutration-mobile.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
```

Comment:
- [ ] [OPTIONAL] I think it could be a good idea to use `background-size: cover;` instead of `contain` because it will occupy the whole section instead of cutting the background image :smiley:

  With `contain`:
  ![image](https://user-images.githubusercontent.com/81522435/143254998-3ef47ceb-c0fb-4573-a253-6217905c5688.png)

  With `cover`:
  ![image](https://user-images.githubusercontent.com/81522435/143255147-62a1003f-f6e5-4c72-af6d-e82427939a11.png)

Reply:
i tried it it looks better with cover